tests/playwright/src/utility/operations.ts

```diff
@@ -214,6 +214,26 @@ export async function deletePodmanMachine(page: Page, machineVisibleName: string
   }
 }
 
+export async function handleResetDefaultConnectionDialog(page: Page): Promise<void> {
+  const connectionDialog = page.getByRole('dialog', { name: 'Podman' });
+  const dialogMessage = connectionDialog.getByText(
+    new RegExp(
+      `.* Podman Machine .+ does not match default connection .+ Do you want to update the default connection?`,
```

Comment:
the dialog message has aria-labels, why are you looking for it in this way, pleasae use the coresponding label.